packages/scene-composer/src/components/panels/scene-components/data-overlay/DataBindingMapEditor.tsx

```diff
@@ -16,6 +18,14 @@ interface IDataBindingMapEditorProps {
   onUpdateCallback: (componentPartial: Partial<IDataOverlayComponentInternal>, replace?: boolean | undefined) => void;
 }
 
+const RemoveButtonContainer = styled.div`
+  float: right;
+  height: 12px;
+`;
+const Spacing = styled.div`
+  height: 8px;
+`;
+
```

Comment:
Question: Where do these magic numbers come from?

Reply:
from manual adjustment to make the component look good